src/main/java/Task.java

```diff
@@ -0,0 +1,59 @@
+import java.time.format.DateTimeFormatter;
+
+
+public class Task {
+    private final Boolean isDone;
+    protected final String name;
+    protected DateTimeFormatter Date;
+    protected DateTimeFormatter Time;
+
+    /**
+     * Constructor for tasks
+     * @param name task name.
+     * @param isDone task status.
+     */
+
+    Task(String name,Boolean isDone){
```

Comment:
I think that some white spaces can be added here 
```suggestion
    Task(String name, Boolean isDone) {
```